app/libs/builders/internship_agreement_builder.rb

```diff
@@ -66,7 +66,7 @@ def preprocess_internship_application_paramns(internship_application)
 
     def preprocess_internship_offer_params(internship_offer)
       {
-        organisation_representative_full_name: internship_offer.employer_name,
+        organisation_representative_full_name: Presenters::User.new(internship_offer.employer).full_name,
```

Comment:
@fitchMitch / @pierrax there was an issue here. it bother me and i'm not confident we can keep this stable without proper testing